### PR TITLE
Start on nginx troubleshooting

### PR DESCRIPTION
**Pull request by ash123456789**
I have started on a very brief troubleshooting guide for nginx, specifically the SSL portion and added recommendations for `ssl_protocols` & `ssl_ciphers` for enabling support on older devices & browsers.

There is a lot more to added but I think this will do for now.

### CODE REVIEW

File: devops/nginx/troubleshooting/ssl.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+## SSL
+
+### Enabling support for older devices & browsers
```
Comment (dextermb) [inline devops/nginx/troubleshooting/ssl.md]: It's probably worth mentioning that `ssl_prefer_server_ciphers` must be set to `on` for `ssl_ciphers` to work correctly:
```
ssl_prefer_server_ciphers on;
```

File: devops/nginx/troubleshooting/ssl.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+You can enable supprot for older devices & browsers by updating `ssl_protocols` and `ssl_ciphers`. `ssl_protocols` define the TLS versions the web server should allow and `ssl_ciphers` defines the supported ciphers.
```
Comment (AlexCatch) [inline devops/nginx/troubleshooting/ssl.md]: `supprot` should be `support`

**Comment by ash123456789** [comment]
@AlexCatch 
> It might also make sense to add that this is not recommended and to try to avoid this at all costs and it adds a vulnerability to the application it's enabled on.

For which part? Updating this part of the configuration and both increase and decrease security and I think I made it clear about which option(s) are not recommended.

**Comment by AlexCatch** [comment]
@ash123456789 Adding ciphers that are old.

I think it should be a bit more clear that it's not a desirable solution at all and that a better solution would always be to convince users to upgrade, it should really be a last resort.

The last thing we need is everyone adding this as a blanket fix for older device support.

**Comment by ash123456789** [comment]
@AlexCatch 
> @ash123456789 Adding ciphers that are old.
> 
> I think it should be a bit more clear that it's not a desirable solution at all and that a better solution would always be to convince users to upgrade, it should really be a last resort.
> 
> The last thing we need is everyone adding this as a blanket fix for older device support.

As far as I am aware the only solutions for the old devices issues are to update the supported cipher suites and drop the minimum supported TLS version to 1.1 or to route the domain through Cloudflare to let their servers handle the requests. TLS v1.1 or higher is fine, anything less than that (e.g. TLS v1, SSL v3) is concerning and highly not recommended. I guess I could make it clearer about why and give reasons though.

**Pull request merged**